### DISCUSSION TITLE
Looking for Arabic language dataset

### DISCUSSION THREAD

**Discussion by TheMShahid** [mrdbourke/pytorch-deep-learning#1113] [General]
Hi everyone i want to train and matching a  model for Arabic language @mrdbourke , kindly suggest me the best resources for the dataset

**Comment by LuluW8071** [comment]
@TheMShahid What type of dataset are you trying to search for? Like 
- Audio clips
- Text Related
- Images for OCR?

**Comment by TheMShahid** [comment]
Thanks for your response, I looking at the dataset for this type of text
(إِنَّ ٱلَّذِينَ كَفَرُوا۟ سَوَآءٌ عَلَيْهِمْ ءَأَنذَرْتَهُمْ أَمْ لَمْ
تُنذِرْهُمْ لَا يُؤْمِنُونَ ٦)

**Reply by LuluW8071** [reply]
You mean? Arabic to English type of dataset for translation?

**Comment by TheMShahid** [comment]
no only Arabic dataset
